sendinblue.php

```diff
@@ -32,16 +32,19 @@
     include_once (_PS_CLASS_DIR_ . '/../classes/Customer.php');
 }
 
-include_once (dirname(__FILE__) . '/sendinblueResources.php');
 if (version_compare(_PS_VERSION_, '1.4.5', '<')) {
+    require(_PS_CLASS_DIR_. '/../modules/sendinblue/Mailin.php');
```

Comment:
dirname(FILE)  not working  for prestashop version 1.4.x, in module list page .
and to make it working for 1.4.x we are using _PS_CLASS_DIR_ for other version we are using as you asked.